examples/rust/opensk/Cargo.toml

```diff
@@ -10,6 +10,7 @@ opensk-lib = { path = "../../../third_party/OpenSK/libraries/opensk", package =
 persistent_store = { path = "../../../third_party/OpenSK/libraries/persistent_store" }
 rand_core = "0.6.4"
 wasefire = { path = "../../../crates/prelude" }
+wasefire-store = { path = "../../../crates/store", features = ["std"] }
```

Comment:
We shouldn't use `std`.

Reply:
I'm not sure what I'm doing wrong here but without the `std` feature, the test throws the error:
```
error[E0432]: unresolved import `crate::Nat`
   --> /usr/local/google/home/suryamidatala/Documents/repos/wasefire/crates/store/src/format/bitfield.rs:20:13
    |
20  | use crate::{Nat, StoreError, StoreResult};
    |             ^^^ no `Nat` in the root
    |
note: found an item that was configured out
   --> /usr/local/google/home/suryamidatala/Documents/repos/wasefire/crates/store/src/lib.rs:407:10
    |
407 | pub type Nat = u32;
    |          ^^^
```

Reply:
I've removed `persistent_store` and `wasefire_stub`. But I'm using impl `CryptoRng` and `RngCore` traits in rng.rs. So shouldn't we need that?

Also I've added `wasefire_error` because we need `Space` and `Code` which I don't think are being re-exported in wasefire.

Reply:
> But I'm using impl `CryptoRng` and `RngCore` traits in rng.rs. So shouldn't we need that?

My bad, I forgot OpenSK uses those traits in its public API. That's expected to depend on `rand_core`, all good.